plugins/daniel-42-z-wallpapershuffler.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "WallpaperShuffler",
+	"capabilities": [
+		"set-wallpaper"
```

Comment:
Not sure about this, 

when i set this on https://github.com/AvengeMedia/dms-plugin-registry/blob/fa1fa959e7858061ed8cae326427b935d51cd4ef/plugins/rochacbruno-worldclock.json#L3 

I was thinking about the DMS capability.

Right now th eplugin system supports only dankbar-widget, and aparently also daemons. (I was not aware daemon was supported)

We need to come with an enum for this property:

- dankbar-widget
- daemon
- timer
- dock-widget
- control-center-widget

Etc...

Open for discussion, we can change it later.